core/src/main/java/org/eclipse/hono/util/TenantObject.java

```diff
@@ -496,4 +496,15 @@ public static JsonObject newAdapterConfig(final String type, final boolean enabl
                 .put(TenantConstants.FIELD_ADAPTERS_TYPE, type)
                 .put(TenantConstants.FIELD_ENABLED, enabled);
     }
+
+    /**
+     * Gets the concurrent connections limit for a tenant.
+     *
+     * @return The concurrent connections limit.
+     */
+    @JsonIgnore
+    public long getConnectionsLimit() {
```

Comment:
in anticipation of the next step (#929) I think that we should make the underlying data structure more generic, i.e. provide for means to add more limits. What about something like:
```
{ "limits": {
      "max-connections": 15000
      }
}
```
which could then be extended later to:
```
{ "limits": {
      "max-connections": 15000,
      "max-bytes-per-month": 20_000_000
      }
}
```
WDYT?

Reply:
👍 I also have the same in mind for the next PR. I can start with this PR.

Reply:
I think it still makes sense to have dedicated getters, e.g. *getConnectionLimit* and (later) *getDataVolumeLimit* or something like that ...

Reply:
@sophokles73 Done. I have pushed the changes.